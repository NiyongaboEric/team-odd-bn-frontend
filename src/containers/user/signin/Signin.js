import React, { Component } from 'react';
import { connect } from 'react-redux';
import Facebook from '../../../views/social/Facebook';
import Google from '../../../views/social/Google';
import '../../../assets/css/style.scss';

export class Signin extends Component {
  render() {
    return (
      <>
        <Google buttonName="log in with Google" />
        <Facebook buttonName="log in with Facebook" />
      </>
    );
  }
}

const mapStateToProps = (state) => ({
});

export default connect(mapStateToProps, null)(Signin);
