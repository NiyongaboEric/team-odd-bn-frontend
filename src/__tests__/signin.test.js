import React from 'react';
import { shallow, mount } from 'enzyme';
import configureMockStore from 'redux-mock-store';
import Signin from '../containers/user/signin/Signin';
import mockData from '../__mocks__/fileMock';

let wrapper; let
  store;
const mockStore = configureMockStore();
const { signinDummyDataWithErrors } = mockData;


describe('Signin testing', () => {
  beforeEach(() => {
    const initialState = signinDummyDataWithErrors;
    store = mockStore(initialState);
    wrapper = shallow(<Signin store={store} />);
  });

  it('Should render class', () => {
    const wrapper2 = mount(<Signin store={store} />);
    expect(wrapper2).toMatchSnapshot();
  });
});
