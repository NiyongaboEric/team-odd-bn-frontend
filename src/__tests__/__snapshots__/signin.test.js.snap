// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Signin testing Should render class 1`] = `ReactWrapper {}`;
